packages/lodestar/package.json

```diff
@@ -82,6 +83,7 @@
     "libp2p-noise": "^1.1.2",
     "libp2p-pubsub": "^0.4.7",
     "libp2p-tcp": "^0.14.6",
+    "lodash.isnil": "^4.0.0",
```

Comment:
This package is literally one line of code:
```ts
function isNil(value) {
      return value == null;
}
```
https://github.com/lodash/lodash/blob/ddfd9b11a0126db2302cb70ec9973b66baec0975/lodash.js#L11973
Do we really have to install a new package for this? Seems like a big stretch

Reply:
my original thinking was that maybe we would need lodash in this package in the future, but then again, that's just a hypothetical.  i'll change it to this for now

Reply:
done